packages/tools/devtools/devtools-view/src/DevtoolsView.tsx

```diff
@@ -250,6 +251,7 @@ function _DevtoolsView(props: _DevtoolsViewProps): React.ReactElement {
 			[ContainerList.MessageType]: async (untypedMessage) => {
 				const message = untypedMessage as ContainerList.Message;
 				setContainers(message.data.containers);
+				setDataObjects(message.data.dataObjects);
```

Comment:
Note: if you make `dataObjects` optional (see my [other comment](https://github.com/microsoft/FluidFramework/pull/24885/files#r2196066955)), then you will want to update this to be:
```suggestion
				setDataObjects(message.data.dataObjects ?? []);
```

That way we always set our local state when we receive a list update, and we won't get an infinite loading spinner for the data objects view.